turbopuffer/__init__.py

```diff
@@ -4,6 +4,8 @@
 api_base_url = os.environ.get('TURBOPUFFER_API_BASE_URL', 'https://api.turbopuffer.com')
 upsert_batch_size = 10_000
 max_retries = 6
+connect_timeout = 10 # seconds
+read_timeout = 180 # seconds
```

Comment:
seems a bit high?

Reply:
agreed, but I'd rather have some wiggle room for users who send gbps of data with a lot of concurrent requests, which can take a long time to process